cmux-tui/bindings/go/client_zero_progress_test.go

```
@@ -0,0 +1,37 @@
+	"io"
+	"net"
+	"testing"
+)
+
+type zeroProgressConn struct{ net.Conn }
+
+func (zeroProgressConn) Write([]byte) (int, error) { return 0, nil }
+
+func TestWriteReturnsNoProgressOnZeroByteWrite(t *testing.T) {
+	clientSide, serverSide := net.Pipe()
+	defer clientSide.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `clientSide.Close` is not checked

(errcheck)